localstack/services/awslambda/lambda_executors.py

```diff
@@ -211,11 +211,24 @@ def __init__(self):
         # locking thread for creation/destruction of docker containers.
         self.docker_container_lock = threading.RLock()
 
+        # On each invocation we try to construct a port unlikely to conflict
+        # with a previously invoked lambda function. This is a problem with at
+        # least the lambci/lambda:go1.x container, which execs a go program that
+        # attempts to bind to the same default port.
+        self.next_port = 0
+        self.max_port = 500
+        self.port_offset = 5000
```

Comment:
Perhaps we could create a constant `LAMBDA_SERVER_PORT_OFFSET = 5000` as a constant on the global scope in this file. At some point we may also want to be able to make it customizable.

Reply:
Will do!